migtests/scripts/live-migration-fallf-run-test.sh

```diff
@@ -218,7 +218,7 @@ main() {
 	step "Clean up"
 
 	./cleanup-db
-	stream_id=$(cat "${EXPORT_DIR}/metainfo/yb_cdc_stream_id.txt")
+	stream_id=$(sqlite3 ${EXPORT_DIR}/metainfo/meta.db "select json_text from json_objects where key='migration_status';" | grep -oP '"YBCDCStreamID":"\K[^"]*')
```

Comment:
Define a function to fetch value of any key from the msr:
```
get_value_from_msr() {
  jq_filter=$1
  val=`sqlite3 ${EXPORT_DIR}/metainfo/meta.db "select json_text from json_objects where key='migration_status';" | jq $jq_filter`
  return $val
}

echo get_value_from_msr('.["YBCDCStreamID"]')
```
Consider the above snippet as a guideline. Do test it before merging.

Reply:
oh, thanks for this suggestion of `jq` tool.